Version.xcconfig

```diff
@@ -7,8 +7,8 @@
 //
 
 // Version [DEFAULT]
-LOOP_MARKETING_VERSION = 0.0.1
```

Comment:
Please revert

Reply:
DIY actually uses this for its version string. Tidepool uses the VersionOverride.